pkg/deploy/devconfig.go

```diff
@@ -0,0 +1,143 @@
+package deploy
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"crypto/x509"
+	"encoding/pem"
+	"io/ioutil"
+	"os"
+
+	"github.com/Azure/go-autorest/autorest/to"
+
+	"github.com/Azure/ARO-RP/pkg/env"
+)
+
+func adminKeyvaultAccessPolicy(_env env.Core) map[string]interface{} {
+	return map[string]interface{}{
+		"objectId": os.Getenv("ADMIN_OBJECT_ID"),
+		"permissions": map[string]interface{}{
+			"certificates": []interface{}{
+				"Get",
+				"List",
+				"Update",
+				"Create",
+				"Import",
+				"Delete",
+				"Recover",
+				"Backup",
+				"Restore",
+				"ManageContacts",
+				"ManageIssuers",
+				"GetIssuers",
+				"ListIssuers",
+			},
+			"secrets": []interface{}{
+				"List",
+			},
+		},
+		"tenantId": _env.TenantID(),
+	}
+}
+
+func deployKeyvaultAccessPolicy(_env env.Core) map[string]interface{} {
+	return map[string]interface{}{
+		"objectId": os.Getenv("AZURE_SERVICE_PRINCIPAL_ID"),
+		"permissions": map[string]interface{}{
+			"secrets": []interface{}{
+				"List",
+				"Set",
+			},
+		},
+		"tenantId": _env.TenantID(),
+	}
+}
+
+func DevConfig(_env env.Core) (*Config, error) {
+	ca, err := ioutil.ReadFile("secrets/dev-ca.crt")
+	if err != nil {
+		return nil, err
+	}
+
+	client, err := ioutil.ReadFile("secrets/dev-client.crt")
+	if err != nil {
+		return nil, err
+	}
+
+	clientCert, err := x509.ParseCertificate(client)
+	if err != nil {
+		return nil, err
+	}
+
+	sshPublicKey, err := ioutil.ReadFile(os.Getenv("HOME") + "/.ssh/id_rsa.pub")
```

Comment:
Would be better of making it env? I personally dont use "this" key for work related stuff, and have separate one. But happy to compromise.